scripts/gatkcondaenv.yml.template

```diff
@@ -29,14 +29,21 @@ dependencies:
 - conda-forge::pytensor=2.18.3
 - conda-forge::scipy=1.11.4
 - conda-forge::h5py=3.10.0
-- conda-forge::pytorch=2.1.0=*mkl*100
+- conda-forge::pytorch=2.3.0=*mkl*100
 - conda-forge::pytorch-lightning=2.4.0   # supports Pytorch >= 2.1 and <= 2.4, used by NVScoreVariants
+- conda-forge::tensorboard=2.8.0
 - conda-forge::scikit-learn=1.3.2
 - conda-forge::matplotlib=3.8.2
 - conda-forge::pandas=2.1.3
 - conda-forge::tqdm=4.66.1
 - conda-forge::dill=0.3.7             # used for pickling lambdas in TrainVariantAnnotationsModel
 - conda-forge::biopython=1.84         # used by NVScoreVariants
+- conda-forge::setuptools>=57.0.0
+- conda-forge::psutil>=5.9.2
+- conda-forge::protobuf<3.20,>=3.9.2  # Protobuf constraint for TensorFlow compatibility
```

Comment:
Didn't we get rid of tensorflow?

Reply:
No idea. Its in the dependencies for the other project:
# Protobuf 4.0 is incompatible with TF. Force < 3.20 until they unblock upgrade.
# See: http://b/182876485
# See: https://github.com/protocolbuffers/protobuf/issues/9954#issuecomment-1128283911
# See: https://cs.opensource.google/tensorflow/tensorflow/+/master:tensorflow/tools/pip_package/setup.py?q=protobuf
protobuf >= 3.9.2, < 3.20